L/LaMEM/build_tarballs.jl

```diff
@@ -24,7 +24,7 @@ mkdir $WORKSPACE/srcdir/lamem/lib/opt
 
 cd $WORKSPACE/srcdir/lamem/src
 export PETSC_OPT=${libdir}/petsc/double_real_Int32/
-ln -s  ${PETSC_OPT}/lib/libpetsc_double_real_Int32.${dlext} ${PETSC_OPT}/lib/libpetsc.${dlext}
```

Comment:
Yes, this wasn't a good idea for several reasons :slightly_smiling_face:

Reply:
yes, sorry...